### DISCUSSION TITLE
请问如何解决：请求的地址无效

### DISCUSSION THREAD

**Discussion by MieMieT** [Lost-MSth/Arcaea-server#95] [Q&A]
【环境搭建】如题，按照使用说明，只更改了主机的地址和启动Link Play服务器
![Q}T KV@I UVNGI_QB1`{_`2](https://user-images.githubusercontent.com/109355245/222973772-35ded80b-62b4-4b71-80f4-322ce970e99a.png)
LinkPlay只更改了`SET_LINKPLAY_SERVER_AS_SUB_PROCESS = False`
萌新求解决方法，提前感谢

**Comment by Lost-MSth** [comment]
如果只是想开服务端，那不需要改任何设置

以及遇到这种有报错的，先百度一下

虽然直接读句子就大致猜到是ip地址写错了（）

**Reply by MieMieT** [reply]
改回配置后服务端可以启动了，谢谢您

**Comment by MieMieT** [comment]
拿到了壳，然后又不会搞了，去百度谷歌油管翻了一遍，没找到合适的指南（大悲
